packages/fx-core/src/component/generator/apiSpec/helper.ts

```diff
@@ -918,6 +930,36 @@ app.ai.action("{{operationId}}", async (context: TurnContext, state: Application
   return "result";
 });
   `,
+  python: `
```

Comment:
This helper.ts file is too huge, in the future, we can move different generators(typeb/sme/ai-generator) to different files.